internal/service/reclaimspace/reclaimspace.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+package service;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/service";
+
+// ReclaimSpace holds the RPC menthod for allowing the
+// communication between the CSIAddons controller and the sidecar.
+service ReclaimSpace {
+    // ControllerReclaimSpace is a procedure that gets called on the CSI
+    // sidecar.
+    rpc ControllerReclaimSpace (ControllerReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
+    // NodeReclaimSpace is a procedure that gets called on the CSI
+    // sidecar.
+    rpc NodeReclaimSpace (NodeReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
+
+  }
```

Comment:
formatting is a little weird, please check indention and add empty lines after a closing `}`